ui/src/app/shared/audit/list/audit.list.component.ts

```diff
@@ -37,6 +38,7 @@ export class AuditListComponent {
             <Column<AuditWorkflow>>{
                 type: ColumnType.CONFIRM_BUTTON,
                 name: '',
+                disabled: !this.canRollback,
```

Comment:
it won't work. Must be in a ngOnInit because it's a variable coming from an @Input